src/cfnlint/data/schemas/other/step_functions/statemachine.json

```diff
@@ -224,12 +228,18 @@
      "pattern": "^.{1,128}$",
      "type": "string"
     },
+    "Output": {
+     "type": "string"
```

Comment:
`Output` can be an object or a string

see some examples on https://docs.aws.amazon.com/step-functions/latest/dg/transforming-data.html#jsonata-examples

Reply:
Update across the board.  Thanks.